spec/features/sign_up_spec.rb

```diff
@@ -0,0 +1,10 @@
+feature 'Sign up' do
+  scenario 'You sign up' do
+    visit('/sign-up')
+    fill_in 'username', with: 'EllieM'
+    fill_in 'email', with: 'makersemail@gmail.com'
+    fill_in 'password', with: 'PASSWORD123'
+    click_on 'submit'
+    expect(page).to have_content 'Peep:'
```

Comment:
I would test for content on the page that signalled a user was signed in. Ex: "What would you like to write for your next peep, EllieM:"